components/ws-daemon/pkg/cpulimit/dispatch.go

```diff
@@ -26,7 +26,7 @@ type Config struct {
 	Enabled        bool              `json:"enabled"`
 	TotalBandwidth resource.Quantity `json:"totalBandwidth"`
 	Limit          resource.Quantity `json:"limit"`
-	BurstLimit     resource.Quantity `json:"bustLimit"`
+	BurstLimit     resource.Quantity `json:"burstLimit"`
```

Comment:
@csweichel heh. I suspect due to this typo bursting might not be working quite right in current gen31 deploy. FYI.

Reply:
The installer also has bustLimit, so it should work. Planning on updating that next.

Reply:
Ah you are right! Thank you!

Reply:
the installer uses the same structure, hence would have the same (incorrect) behaviour